### ISSUE TITLE
Adding custom functions around moveable element

### ISSUE THREAD

**Issue by webAravis** [daybrush/moveable#335]
## Environments
* Framework name: Angular
* Framework version: 10
* Moveable Component version: 0.15.4
* Testable Address(optional):

## Description
Hello and thanks for this great package. Is there a way to add custom functions with their own markup, css and behavior like a custom delete icon on this post #246 ?

**Comment by daybrush** [comment]
@webAravis 

What does setTransform contain?

Translate => Rotate => Is it in order of Scale?

**Comment by daybrush** [comment]
Let me check the second problem.

**Comment by daybrush** [comment]
@webAravis 
However, custom props cannot be used in angular, and css should be set to global. or use inline css.

```js

customAble = {
   name: "customAble",
   always: true,
   render(moveable, React) {
    return React.createElement(...);
   },
};

[ables]="[customAble]"
```

**Comment by daybrush** [comment]
@webAravis 
I wrote a bit of documentation.
https://github.com/daybrush/moveable/blob/master/packages/react-moveable/src/react-moveable/ables/README.md

**Comment by lixue007** [comment]
![demo](https://ftp.bmp.ovh/imgs/2020/10/532be9d8aef9cdf4.gif)

**Comment by webAravis** [comment]
Thank you, I'll try it and tell you!

**Comment by negue** [comment]
Does it mean you need React renderer inside Angular to add custom ables to your moveables ?

**Comment by daybrush** [comment]
@negue 

Yes. However, it does not import react. Use React as the second argument.


```js
customAble = {
   name: "customAble",
   always: true,
   props: {},
   events: {},
   render(moveable, React) {
    return React.createElement(...);
   },
};
```

**Comment by iprasadg** [comment]
@daybrush How to use [ables] in Angular?

**Comment by iprasadg** [comment]
@daybrush @webAravis How to use [ables] in Angular? If you have a example in angular please share, it will be great help. 

Thanks in advance

**Comment by zarausto** [comment]
If you can please share examples of using Custom Able with vanilla would be great =)

**Comment by born2net** [comment]
any updates on this, posted here too: 
https://github.com/daybrush/moveable/issues/1007
